src/site/content/en/angular/accessible-angular-with-codelyzer/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: 'Audit your Angular app's accessibility with codelyzer'
```

Comment:
Need to use double quotes here because there's a single quote contained within. Otherwise it breaks the YAML parser.

Reply:
Fixed